tiatoolbox/annotation/storage.py

```diff
@@ -1628,6 +1628,71 @@ def _append(self, key: str, annotation: Annotation, cur: sqlite3.Cursor) -> None
             token,
         )
 
+    @staticmethod
+    def __initialize_query_string_parameters(
```

Comment:
This shouldn't be a double underscore

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/330/commits/82155397526452234dc3fbbfa80a77e520506d13